prog-o-meter.py

```diff
@@ -51,48 +52,54 @@ def __init__(self, _days, _filename, _username):
         self.days = _days
         self.GOAL = 100
         self.rectangle_list = []
+        self.encourage = ["Nice Job!", "The force is strong with you!", "Well Done!", "You are doing it!", "Never, never, never give up.", "If you dream it, you can do it.", "Everything you can imagine is real.", "Hope is a waking dream.", "You are the best!", "Always believe in yourself."]
         self.days_remaining = self.GOAL - self.days
         self.completion_date = self.get_completion_date(self.days_remaining-1)
-        # Tkinter instantiation 
+        self.x = 0
+        # Tkinter instantiation
         self.canvas_layout()
         self.button_layout()
         self.prog_o_meter()
         self.progress()
         self.root.mainloop()
     def canvas_layout(self):
         """Display a Tkinter canvas.
-        
-        Creates a 1200x600 px canvas, with a greeting text, and a text stating how many days the user have left untill they reach the goal.  
-       
+
+        Creates a 1200x600 px canvas, with a greeting text, and a text stating how many days the user have left untill they reach the goal.
+
         Attributes:
             CANVAS_WIDTH: The width of the canvas is hardcoded to 1200 px.
             CANVAS_HEIGHT: The height of the canvas is hardcoded to 600 px.
             canvas: The Tkinter canvas widget.
-            countdown_text: A text element on the canvas, stating how many days the user have left before reaching their goal. 
+            countdown_text: A text element on the canvas, stating how many days the user have left before reaching their goal.
         """
         self.CANVAS_WIDTH = 1200
         self.CANVAS_HEIGHT = 600
         VERTICAL_TEXT_POSITION = 100
         self.canvas = Tk.Canvas(self.root, width = self.CANVAS_WIDTH, height = self.CANVAS_HEIGHT)
         self.canvas.pack()
-        self.canvas.create_text(self.CANVAS_WIDTH/2, VERTICAL_TEXT_POSITION, text = ("".join(("Hello ", self.username))))
+        self.hog = self.canvas.create_text(self.CANVAS_WIDTH/2, VERTICAL_TEXT_POSITION, text = ("".join(("Hello ", self.username))))

```

Comment:
Again, to make it easy for people to read the code, giving explanatory names is always good. I am not sure what hog means?  Could we call the variable something like self.greeting?